utility/mm2_update-master-directory-list

```diff
@@ -634,8 +743,111 @@ def sync_directories_from_disk(
     sync_category_directories(
         session, config, diskpath, category, category_directories)
 
+    if category.delete_directories:
+        nuke_gone_directories(session, diskpath, category)
+
+
+def sync_directories_from_fullfiletimelist(
```

Comment:
I recommend documenting the purpose, arguments, and return value(s) of this function